app/controllers/shifts_controller.rb

```diff
@@ -88,4 +93,10 @@ def current_org_id
   def shift_params
     params.require(:shift).permit(:shift_open, :shift_role, :shift_description, :shift_start, :shift_end, :shift_pay, :organization_id, :worker_id)
   end
+
+  def shift_owner
```

Comment:
Hey @saradotlog, there is a cleaner, more Rails-esque way to define this method 😄 

```ruby
def shift_owner?
    @shift.organization_id == current_org_id
end
```

Adding the question mark to the end of the method indicates that the method returns a `true/false`.
In Rails, the last evaluation is automatically returned, so no need for the `if` block 😉 

Question: I looked for where `current_org_id` is defined in the codebase but I couldn't find it. Am I missing something?

Reply:
Hi @msarit Thanks a lot for the tip! I actually looked up time ago the difference between a method with and without a "?" but I totally forgot about it, this is a good working example.
Then I reckon we should do the same for the is_owner method?

Reply:
About the `current_org_id` is defined in this same file, in line 88:
```
def current_org_id
    puts "\n\n\nCurrent User ID: #{current_user.id}\n\n\n"
    return Organization.where(user_id: current_user.id).pluck(:id).first
end
```
But it was already there, I didn't define it. I guess I should remove the "puts..." statement as it was for debugging?